imgfac/Builder.py

```diff
@@ -472,3 +475,19 @@ def _delete_image(self, provider, credentials, target, image_object, parameters)
         finally:
             # We only shut the workers down after a known-final state change
             self._shutdown_callback_workers(image_object, self._deletion_cbws)
+
+    # defined here as Koji imports builder classes directly
+    def signal_handler(self, signum, stack):
+        if signum == signal.SIGINT:
+            # Run the abort() method in any running target builder
+            logging.warn('caught signal SIGINT, stopping...')
```

Comment:
Shouldn't you use `self.log` here, as the rest of the code in this class does?

Reply:
Nice catch, thanks